### ISSUE TITLE
 Fatal error: Uncaught exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'phpmixbillv50.tbl_users' doesn't exist' in C:\wamp64\www\phpmixbillv50\system\orm.php on line 435

### ISSUE THREAD

**Issue by danielbaricawa** [ibnux/phpmixbill#12]
( ! ) Fatal error: Uncaught exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'phpmixbillv50.tbl_users' doesn't exist' in C:\wamp64\www\phpmixbillv50\system\orm.php on line 435
( ! ) PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'phpmixbillv50.tbl_users' doesn't exist in C:\wamp64\www\phpmixbillv50\system\orm.php on line 435
Call Stack
#	Time	Memory	Function	Location
1	0.0008	239304	{main}( )	...\index.php:0
2	0.0032	298864	require( 'C:\wamp64\www\phpmixbillv50\system\boot.php' )	...\index.php:11
3	0.0767	1942776	include( 'C:\wamp64\www\phpmixbillv50\system\controllers\admin.php' )	...\boot.php:193
4	0.0772	1947016	ORM->find_one( )	...\admin.php:21
5	0.0773	1947096	ORM->_run( )	...\orm.php:613
6	0.0790	1947400	ORM::_execute( )	...\orm.php:1834
7	0.0794	1943512	execute ( )	...\orm.php:435

can anyone here help me? I really want to know what's the in here.

![untitled](https://user-images.githubusercontent.com/43360291/45661328-a3288980-bb2f-11e8-8786-dd77b2ac1f3b.png)

**Comment by aredan** [comment]
How are you doing the install?  
 When I did it, it fails with the same error. try it manually and found that the SQL schema has an error on the timestamp that is using and that is killing the installer and some DB tables aren't being created.

**Comment by aredan** [comment]
I found what I did to get the application working.
run the installer from the web and them remove the DB create via de web installer, create a new one with the same name.
Edit the phpmixbill.sql file from the installer:
change line 92

1. from `date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
2. to     `date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,


Basically, you are just changing the default date format for that table.
